src/AcmAutoRefreshSelect/AcmAutoRefreshSelect.tsx

```diff
@@ -0,0 +1,163 @@
+import React, { useEffect, useState } from 'react'
+import { SyncAltIcon } from '@patternfly/react-icons'
+import { Dropdown, DropdownItem, DropdownToggle } from '@patternfly/react-core'
+import { makeStyles } from '@material-ui/styles'
+
+const DEFAULT_REFRESH_TIME = 60
+
+export type AcmAutoRefreshSelectProps = {
+    refetch: () => void
+    refreshValues: number[]
+    refreshCookie: string
+    pollInterval: number
+    startPolling: (pollInterval: number) => void
+    stopPolling: () => void
```

Comment:
Do we really need `startPolling` and `stopPolling`?  It seems simpler to use only `refetch`.

Reply:
I would think not - will try and work those out of component